lib/jekyll/commands/new.rb

```diff
@@ -57,6 +74,12 @@ def initialized_post_name
 
         private
 
+        def sanitized(args)
```

Comment:
The reason is, because I **felt** this method to be faster. But if your expertise says its wrong, I'll look at standard library.